### PR TITLE
feat(frontend): Improved footer styles #471

### LINKED ISSUE #471: Improve footer styles
As a user  i want to se diferent  types styles so that i can create curiosit
https://guide.eoscostarica.io/docs/eos-learn/eos-blockchain

image

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?

- resolves #471 
Note: react-icons was installed: npm install react-icons --save or yarn install react-icons

### Steps to test
1. Navigate to the bottom of [guide.eosio](https://guide.eoscostarica.io/) and see page footer

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Comment by fvives99** [comment]
image

**Comment by fvives99** [comment]
image

File: src/theme/Footer/index.js [hunk 1]
```diff
@@ -5,128 +5,202 @@
...
-      </div>
-    </footer>
-  );
-}
-
-export default Footer;
+ import React, { useEffect, useState } from 'react'
+ import clsx from 'clsx';
+ 
+ //import Link from '@docusaurus/Link';
```
Comment (AngeloCG97) [inline src/theme/Footer/index.js:11]: Please remove this comment

File: src/theme/Footer/index.js [hunk 2]
```diff
@@ -5,128 +5,202 @@
...
+ import useBaseUrl from '@docusaurus/useBaseUrl';
+ import styles from './styles.module.css';
+ import whiteLogo from '../../../static/img/eoscostarica-logo.png'
+ import blackLogo from '../../../static/img/eos-logo-dark.png'
+ import { Box, Grid, Container, Link, Image } from '@material-ui/core';
+ import { AiFillGithub, AiFillLinkedin, AiOutlineTwitter, AiOutlineInstagram, AiFillYoutube } from "react-icons/ai";
+ import { BsMedium } from "react-icons/bs";
+ 
+ 
+
```
Comment (AngeloCG97) [inline src/theme/Footer/index.js:23]: Please reduce spaces

File: src/theme/Footer/index.js [hunk 3]
```diff
@@ -5,128 +5,202 @@
...
+                 <a href={logo.href}>
+                   <FooterLogo alt={logo.alt} url={logoUrl} />
+                 </a>
+                 {copyright ? <div>Copyright &reg; {new Date().getFullYear()} EOS Costa Rica</div> : null}
+ 
+               </Box>
+             </Grid>
+ 
+             <Grid item xs={12} sm={6}>
+               <Box border={1} borderTop={0} borderBottom={0} borderColor={'#BDBDBD'} paddingLeft={20}>
```
Comment (AngeloCG97) [inline src/theme/Footer/index.js:107]: Unecessary curly brases in borderColor={'#BDBDBD'}

**Comment by AngeloCG97** [comment]
> image

Please paste screen mobile version

**Comment by AngeloCG97** [comment]
Please resolve conflict

File: src/theme/Footer/index.js [hunk 4]
```diff
@@ -5,128 +5,162 @@
...
+   const { copyright, logo = {} } = footer || {};
+   const [logoUrl, setLogoUrl] = useState(useBaseUrl(logo.src));
+   const { colorMode } = useColorMode();
+ 
+    if (!footer) {
+     return null;
+   }
+ 
+   useEffect(() => {
+     console.log(isDarkTheme)
```
Comment (AngeloCG97) [inline src/theme/Footer/index.js:40]: Please move this comment

Comment (fvives99) [reply src/theme/Footer/index.js:40]: What comment?

Comment (fvives99) [reply src/theme/Footer/index.js]: removed

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Review by xavier506 — CHANGES_REQUESTED** [review]
@fvives99 

Are you able to run `yarn build` on this branch?

**Comment by fvives99** [comment]
> @fvives99
> 
> Are you able to run `yarn build` on this branch?

Will fix asap

**Comment by fvives99** [comment]
> > image
> 
> Please paste screen mobile version

image

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**